### DISCUSSION TITLE
能不能让某个域名强制走代理

### DISCUSSION THREAD

**Discussion by yintao-xu** [v2fly/v2ray-core#1871] [Q&A]
在windows系统下，用的v2raycore中的绕过大陆模式。
访问www.tiktok.com会被认为访问的是国内网址，但是不走代理是访问不了的，在全局模式下可访问。
要怎么设置才能使www.tiktok.com在绕过大陆模式下能访问？

**Comment by yintao-xu** [comment]
已解决，在路由最前面新增一条proxy规则精确匹配域名。
![](https://s3.bmp.ovh/imgs/2022/07/15/e7fbd018a83aaf48.png)

**Reply by Veankeelin** [reply]
好像不行啊，你输入的事TK前台的，我广告那边进不去，按你的输入了也进不去
